pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -57,6 +57,8 @@ let
   # use compatible cuDNN (https://www.tensorflow.org/install/source#gpu)
   # cudaPackages.cudnn led to this:
   # https://github.com/tensorflow/tensorflow/issues/60398
+  # NOTE: Make sure to update the broken condition in `meta` when there is a
+  # cuDNN version bump.
```

Comment:
Maybe `let cudnnAttribute = "cudnn_8_6"` and `hasAttr` then

Reply:
That is cleaner; fixed in the latest!